src/testshade/testshade.cpp

```diff
@@ -390,15 +390,7 @@ action_groupspec (int argc, const char *argv[])
     if (OIIO::Filesystem::exists (groupspec)) {
         // If it names a file, use the contents of the file as the group
         // specification.
-        std::ifstream in;
-        OIIO::Filesystem::open (in, groupspec);
-        groupspec.clear();
-        while (in.good()) {
-            std::string line;
-            std::getline (in, line);
-            groupspec += line + "\n";
-        }
-        in.close ();
+        OIIO::Filesystem::read_text_file (groupspec, groupspec);
```

Comment:
It seems a bit dangerous to send in a string view to a string that is being mutated (though the implementation details of this function make it ok I think).